src/powershell/Public/ConvertTo-FinOpsSchema/ConvertTo-FinOpsSchema.ps1

```diff
@@ -0,0 +1,221 @@
+                    throw "Cost details schema not supported."
+                }
+            # else throw an error 
+            } else {
+                throw "Combined Cost data is empty or null."
+            }
+            # TODO: Filter the actual cost data to rows where ChargeType == "Purchase" and PricingModel == "Reservation" or "SavingsPlan" (leave amortized cost as-is).
+            # This to ensure that we only have rows with Purchase, reservation or savings plan data. Only applicable to actual cost data. Leaving the amortized cost data as-is.
+
+            # 1. Uppercase Column Names Modifications
+            $CombinedCost = $CombinedCost | ForEach-Object {
+                $obj = $_
+                $newObj = New-Object PSObject
+                foreach ($property in $obj.PSObject.Properties) {
+                    # If property name starts with a lowercase letter, modify the name
+                    $newPropertyName = if ($property.Name -cmatch "^[a-z]") {
+                        $property.Name -creplace '^[a-z]', { $_.Value.ToUpper() }
+                    } else {
+                        # If not, keep the original name
+                        $property.Name
+                    }
+                    # Add the property to the new object with the potentially modified name
+                    $newObj | Add-Member -MemberType NoteProperty -Name $newPropertyName -Value $property.Value
+                }
+                # Return the new object with modified property names
+                $newObj
+                # write-host "newObj: $($newObj | Select-Object -First 1 | Out-String)"
+            }
+            
+            # Write-Host "Sample data after Uppercase Column Name Modifications: $($CombinedCost | Select-Object -First 1 | Out-String)"
+            # Count the number of rows in the data
+            Write-Host "Combined Cost Data Count after Uppercase Column Name Modifications: $($CombinedCost.Count)"
+
+
+            # 2. Column Name Modifications (prepend 'ftk_')
+            $CombinedCost = $CombinedCost | ForEach-Object {
+                $properties = @{}
+                $_.PSObject.Properties | ForEach-Object {
+                    $newPropertyName = "ftk_" + ($_.Name.Substring(0,1).ToUpper() + $_.Name.Substring(1))
+                    $properties.$newPropertyName = $_.Value
+                }
+                New-Object PSObject -Property $properties
+            }
+            # Write-Host "Sample data after Column Name Modifications: $($CombinedCost | Select-Object -First 1 | Out-String)"
+            Write-Host "Combined Cost Data Count after Column Name Modifications: $($CombinedCost.Count)"
+
+            # 3. Column Additions (ftk_AccountName, ftk_AccountId, ftk_AccountOwnerId, ftk_AvailabilityZone, ftk_EffectivePrice)
+            $CombinedCost = $CombinedCost | ForEach-Object {
```

Comment:
I'm starting to wonder if we should loop thru each row and do a column-to-column mapping 🤔 That way, we only run thru it once and copy an old value to a new column name and are done in one shot. Thoughts?

In this case, these are columns that may not be there that we're adding to normalize. So we'd have something like: 
```powershell
$newObject = ${
    ftk_AccountId = $oldRow.AccountOwnerId
    ftk_AccountName = $oldRow.AccountName
    ...
}
```